src/components/Timer/Timer.scss

```diff
@@ -149,80 +155,48 @@ $timer__progress-bar-color: #6096ff;
   width: $timer__short-actions-height;
 }
 
-.short-action__button:hover {
-  background-color: $timer__progress-bar-color;
-  color: $color-white;
-}
-
-.short-action__tooltip,
-.increment-button__tooltip {
-  width: max-content;
-  visibility: hidden;
-  opacity: 0;
-  transition: opacity 150ms linear;
-  position: absolute;
-  left: 50%;
-  transform: translateX(-50%);
-  border-radius: $rounded--full;
-  padding: $spacing--xs $spacing--base;
-  background-color: $tooltip-background-color--light;
-  color: $menu-icon-background-color--dark;
-  font-size: $text--base;
-  font-weight: bold;
-  bottom: calc(100% + 6px);
-  box-shadow:
-    0 1px 3px 0 rgb(0 0 0 / 0.1),
-    0 1px 2px -1px rgb(0 0 0 / 0.1);
-  // Increased the z-index to make sure the tooltip doesn't get overlapped by the absolute positioned icon
-  z-index: calc($infobar-z-index + 50);
-}
-@media (prefers-reduced-motion) {
-  .short-action__tooltip {
-    transition: none;
-  }
-}
-
-.timer__increment-button:hover > .increment-button__tooltip,
-.short-actions__short-action:hover > .short-action__tooltip {
-  visibility: visible;
-  opacity: 1;
-}
-
-.increment-button__tooltip::after,
-.short-action__tooltip::after {
-  content: "";
-  position: absolute;
-  border: 5px solid;
-  border-color: $tooltip-background-color--light transparent transparent transparent;
-  left: 50%;
-  transform: translateX(-50%);
-  top: 100%;
-}
-
 [theme="dark"] {
   .timer {
     color: $color-white;
     background-color: $menu-icon-background-color--dark;
     border-color: $menu-icon-background-color--dark;
-  }
 
-  .timer__increment-button:focus-within {
-    outline: 2px solid $color-planning-pink;
+    &:has(:focus-visible) {
+      outline: 2px solid $color-planning-pink;
```

Comment:
It feels a bit weird, personally, to have a blue progressbar with a red focus outline on darkmode.

Additionally, the progressbar color is twice defined with the same hex code (`$vote-display__progress-bar-color` and `$timer__progress-bar-color`), however, this PR did not affect this.

Reply:
I was inspired by other elements that also use our Planning Pink as a focus colour in dark mode. I wanted to remain consistent.